graph/patterns/change-tracking.md

```diff
@@ -0,0 +1,163 @@
+  - The change records are full or partial representations of the resources according to their resource types.
+  - When a change representing a resource update is included in the payload the API producer MAY return either the changed properties or the full entity. The ID of the resource MUST be included in every change record.
+  - When an entity is deleted, the delta function MUST return the ID of the deleted entity as well as an `@removed` annotation with the reason field.
+  - When an entity is deleted, the reason MUST be set to “changed” if the entity can be restored.
+  - When an entity is deleted. the reason MUST be set to “deleted” if the entity cannot be restored.
+  - There is no mechanism to indicate that a resource has entered or exited the dataset based on a change that causes it to match or no longer match any `$filter` query parameter.
+  - When a link to an entity is deleted, when the linked entity is deleted, or when a link to an entity is added, the implementer MUST return a `property@delta` annotation. 
+  - When a link to an entity is deleted, but the entity still exists, the reason MUST be set to `changed`.
+  - When a link to an entity is deleted along with the entity, the reason MUST be set to `deleted`.
+  
+API producers MAY choose to collate multiple changes to the same resource into a single change record. 
+
+## When to use this pattern
+
+API consumers want a pull mechanism to request and process change to Microsoft Graph data, either via proactive polling or by responding to Microsoft Graph notifications.
+
+API consumers need guaranteed data integrity over the set of changes to Microsoft Graph data.
+
+## Considerations
+
+ - API service MAY be able to respond to standard OData query parameters with the initial call to the delta function:
+
+    - `$select` to enforce the set of properties on which change is reported.
+    - `$filter` to influence the scope of changes returned.
+    - `$expand` to include linked resources with the set of changes.
+    - `$top` parameter to influence the size of the set of change records.
+  
+    These query parameters MUST be encoded into subsequent `@odata.nextLink` or `@odata.deltaLink`, such that the same options are preserved through the call sequence without callers respecifying them, which MUST NOT be allowed. OData query parameters must be honored in full, or a 400-error returned.
+- Making a sequence of calls to a delta function followed by the opaque URLs in the `nextLink` and `deltaLink` MUST guarantee that the data at the start time of the call sequence and all changes to the data thereafter will be returned at least once. It is not necessary to avoid duplicates in the sequence. When the delta function is returning changes, they MUST be sequenced chronologically refer to [public documentation](https://learn.microsoft.com/en-us/graph/delta-query-overview?view=graph-rest-1.0) for more details.
+- The delta function can be bound to a collection, as with
```

Comment:
I can also be bound to a function that returns an entity collection.  See [Pull Request 7242963](https://dev.azure.com/msazure/One/_git/AD-AggregatorService-Wiki/pullrequest/7242963): Add info on change tracking for functions, for an example, including the schema updates.

Reply:
Actually the "some logical parent resource" phrase allows for a function but i also added an example and formatting to highlight options.